invite.js

```diff
@@ -30,7 +34,8 @@ exports.create = function (api) {
     // connect to the remote pub using the invite code
     ssbClient(null, {
       remote: data.invite,
-      manifest: { invite: {use: 'async'}, getAddress: 'async' }
+      manifest: { invite: {use: 'async'}, getAddress: 'async' },
+      appKey: config.caps.shs
```

Comment:
arr flip. I forgot this and published. I'm gonna go in and patch it now. sorry D: